src/pages/Profile/index.js

```diff
@@ -23,90 +32,361 @@ import avatar from '../../assets/avatar.svg';
 class Profile extends React.Component {
   constructor(props) {
     super(props);
+    this.ref = {
+      name: React.createRef(),
+      email: React.createRef(),
+      position: React.createRef(),
+    };
     this.state = {
-      isChecked: true,
+      dataExpireValue: '6 months',
+      editView: false,
+      displayName: 'Anisha Swain',
+      emailVal: 'xyz.redhat@gmail.com',
+      positionVal: 'Associate Software Engineer',
+      accountType: 'Pbench',
+      accountStatus: 'Active',
+      color: '#0080005e',
     };
-  }
 
-  componentDidMount = () => {};
+    this.dataExpireOptions = [
+      { value: '2 months', label: '2 months', disabled: false },
+      { value: '6 months', label: '6 months', disabled: false },
+      { value: '1 year', label: '1 year', disabled: false },
+    ];
+
+    this.accountOptions = [
+      { value: 'Pbench', label: 'Pbench', disabled: false },
+      { value: 'Google', label: 'Google', disabled: false },
+      { value: 'RedHat SSO', label: 'RedHat SSO', disabled: false },
+      { value: 'Github', label: 'Github', disabled: false },
+    ];
+
+    this.accountStatus = [
+      { value: 'Active', label: 'Active', disabled: false },
+      { value: 'Inactive', label: 'Inactive', disabled: false },
+    ];
+
+    this.ondataExpireValueChange = dataExpireValue => {
+      this.setState({ dataExpireValue });
+    };
 
-  handleChange = isChecked => {
-    this.setState({ isChecked });
-  };
+    this.onaccountStatusChange = accountStatus => {
+      this.setState({
+        accountStatus,
+        color: accountStatus === 'Active' ? '#0080005e' : '#ff000052',
+      });
+    };
+
+    this.edit = () => {
+      const { editView } = this.state;
+      this.setState({
+        editView: !editView,
+      });
+    };
+
+    this.saveEdit = () => {
+      const { editView, displayName, emailVal, positionVal } = this.state;
+      const { name, email, position } = this.ref;
+
+      const editedName = name.current.value === '' ? displayName : name.current.value;
+      const editedEmail = email.current.value === '' ? emailVal : email.current.value;
+      const editedPosition = position.current.value === '' ? positionVal : position.current.value;
+
+      this.setState({
+        editView: !editView,
```

Comment:
`editView: !editView,` => `!editView`

Reply:
Need to put the hardcoded value in the mock file. Converting the PR into the draft for the same.

Reply:
`Parsing error: Unexpected token`